project_event/models/project_task.py

```diff
@@ -329,13 +329,15 @@ def create_main_task(self, vals, parent_id):
 
     def is_new_task(self, vals):
         if 'activity_task_type' in vals:
-            return vals['activity_task_type'] == 'task'
+            vals['activity_task_type'] == 'task'
+            return True
```

Comment:
It will always return true in this case. we dont need to refactor this code

Reply:
@rimbendhaou  I know but method signature has to be consistent